package-res/resources/web/pentaho/data/_filter/AbstractFilter.js

```diff
@@ -79,9 +79,20 @@ define([
       return _apply(this, dataTable);
     },
 
-    transform: function(iteratee){
+    /**
```

Comment:
Since the idea here is to return a transformed version of a node (tranversing the tree if need be), I'll change the method's name to "visit"